src/common/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, Input } from "../Button/";
+import { dictionary } from "../../strings";
```

Comment:
Create i18n folder for the dictionary
Create path aliases. Something like this: `@components` `i18n`